pkg/azurefile/azurefile.go

```diff
@@ -1032,12 +1033,14 @@ func (d *Driver) ResizeFileShare(ctx context.Context, subsID, resourceGroup, acc
 			if err != nil {
 				return true, err
 			}
+			if ptr.Deref(fileShare.FileShareProperties.ShareQuota, 0) >= int32(sizeGiB) {
+				klog.Warningf("file share size(%dGi) is already greater or equal than requested size(%dGi), accountName: %s, shareName: %s",
```

Comment:
so, if `fileShare.FileShareProperties.ShareQuota` was nil, then IMO the logging statement below should have also guarded against printing nil pointer. But that is perhaps a minor nit. :/

Reply:
https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/2433 would fix the issue, thx